Archipelago.RetroArchClient/OcarinaOfTime/Services/LocationCheckService.cs

```
@@ -88,7 +88,7 @@ await memoryService.WriteByteArray(
 
         var locationsToCheck
             = AllLocationInformation
-                .AllLocations
+                .AllLocations!
                 .Where(location => !_areasToSkipChecking.Contains(location.Area))
```

If this is nullable then it's better to check for null before this point and throw a descriptive exception rather than just silencing the warning.

Although looking at this, I see no reason why this should be nullable, so the better solution here is making AllLocations not a nullable list since it has no reason to be one.